src/app.config.js

```diff
@@ -363,7 +363,7 @@ appConfig.httpErrorWhitelist = [
   { status: 403, detail: 'Forbidden.' },
   { status: 404, detail: 'Not found.' }
 ];
-appConfig.defaultCacheExpire = 1; // in minutes
+appConfig.defaultCacheExpire = 60000; // in minutes
```

Comment:
Is this really what we want for the default? Or have the units changed?

Reply:
I'll update comment, it's in `ms` now